packages/mask/src/manifest.json

```diff
@@ -10,6 +10,6 @@
     "browser_action": { "default_popup": "popups.html" },
     "homepage_url": "https://mask.io",
     "description": "The portal to the new & open Internet. Send encrypted message and decentralized Apps right on top of social networks.",
-    "web_accessible_resources": ["js/*", "*.png", "*.css"],
+    "web_accessible_resources": ["js/*", "*.svg", "*.png", "*.css"],
```

Comment:
CC @Jack-Works Does it break auto-review?

Reply:
Nope. Don't worry